apps/rush/src/utilities/PublishUtilities.ts

```diff
@@ -139,7 +150,7 @@ export default class PublishUtilities {
   }
 
   public static isRangeDependency(version: string): boolean {
-    const LOOSE_PKG_REGEX: RegExp = />=?(?:\d+\.){2}\d+\s+<(?:\d+\.){2}\d+/;
+    const LOOSE_PKG_REGEX: RegExp = />=?(?:\d+\.){2}\d+(\-[0-9A-Za-z-.]*)?\s+<(?:\d+\.){2}\d+/;
```

Comment:
The logic here is only for comparator range, not for all types of ranges. We can improve our range support in the next improvement. 

---
In reply to: [134069328](https://github.com/Microsoft/web-build-tools/pull/277#discussion_r134069328) [](ancestors = 134069328)